src/components/organization/create-dho-widget.vue

```diff
@@ -8,7 +8,7 @@ export default {
 </script>
 
 <template lang="pug">
-      widget.q-my-md.position-relative(title="Create your DHO")
-        .text-ellipsis.text-grey-7.q-my-xl Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-        q-btn.q-mt-xl.q-px-lg(rounded color="primary" no-caps @click="$router.push({ name: 'dho-creation' })") New DHO
+widget.q-my-md.position-relative(title="Create your DHO")
+  .text-ellipsis.text-grey-7.q-my-xl Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
```

Comment:
@arsenijesavic We shouldn't use .text-grey classes, only the ones defined on the quasar.variables.styl, in this case, I think it would be .text-body